caio/linux_aio.c

```diff
@@ -964,6 +964,25 @@ PyMODINIT_FUNC PyInit_linux_aio(void) {
         return NULL;
     }
 
+    aio_context_t temp_ctx;
```

Comment:
For > `0.9.3`, we're getting the `Error on io_setup with code 22` `ImportError` (while working fine in other Linux environments). For this `EINVAL`, [man 2 io_setup](https://man7.org/linux/man-pages/man2/io_setup.2.html) reads:

> ctx_idp is not initialized, or the specified nr_events exceeds internal limits. nr_events should be greater than 0.

And indeed, there might be no guarantee that `temp_ctx` is initialized here - in contrast to the "production code" in `AIOContext_init`. Explicitly also setting it to `0` might thus be advisable in any case.

[fs]